specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -567,6 +790,117 @@
         }
       },
       "description": "The information of an AvailablePrivateEndpointType."
+    },
+    "PrivateDNSZoneGroup": {
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateDNSZoneGroupProperties",
```

Comment:
PrivateDnsZoneGroupPropertiesFormat

Reply:
I've renamed it to PrivateDnsZoneGroupPropertiesFormat, will push once I've resolved rest of comments.